licensing.md

```diff
@@ -0,0 +1,27 @@
+# Licensing
+
+## Each repository should include a licence file
+
+This should be called `LICENCE` or `LICENCE.md`. "License" is the U.S. English spelling.
+
+GitHub.com will still show licence details for the British English spelling.
+
+## Use MIT
+
+At GDS we use the [MIT License](https://opensource.org/licenses/MIT).
```

Comment:
@willp-bl I prefer the original. For edge cases I'd expect those conversations to happen anyway. When someone has good cause for an exception, this guidance could be updated.